pootle/apps/pootle_store/models.py

```diff
@@ -1280,16 +1280,27 @@ def create(self, *args, **kwargs):
         if "filetype" not in kwargs:
             filetypes = ProjectFiletypes(kwargs["translation_project"].project)
             kwargs['filetype'] = filetypes.choose_filetype(kwargs["name"])
+        tp = kwargs["translation_project"]
+        if tp.language == tp.project.source_language:
+            kwargs["is_template"] = True
         kwargs["pootle_path"] = (
             "%s%s"
             % (kwargs["parent"].pootle_path, kwargs["name"]))
         return super(StoreManager, self).create(*args, **kwargs)
 
     def get_or_create(self, *args, **kwargs):
         store, created = super(StoreManager, self).get_or_create(*args, **kwargs)
+        update = False
+        if created:
+            tp = store.translation_project
+            if tp.language == tp.project.source_language:
+                store.is_template = True
+                update = True
         if created and "filetype" not in kwargs:
             filetypes = ProjectFiletypes(store.translation_project.project)
             store.filetype = filetypes.choose_filetype(store.name)
+            update = True
+        if update:
```

Comment:
im not sure this is a big issue at Store creation time to set a foreign key, and really no different to being activated in Store.save

It might be possible to set the kwargs similar to how it is done in create, but tbh i think this makes it more complex